client/my-sites/stats/activity-log/unavailability-notice.jsx

```diff
@@ -0,0 +1,60 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import { getSiteAdminUrl } from 'state/sites/selectors';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getRewindState } from 'state/selectors';
+
+export const UnavailabilityNotice = ( { adminUrl, reason, rewindState, slug, translate } ) => {
+	if ( rewindState !== 'unavailable' ) {
+		return null;
+	}
+
+	switch ( reason ) {
+		case 'no_connected_jetpack':
+			return (
+				<Banner
+					icon="history"
+					href={ adminUrl }
+					title={ translate( 'The site is not connected' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'The site is not connected.' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/rewind/show-unavailability-report)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).